### PR TITLE
Adding two features: Change the sound volume and change playlist

### PR DESCRIPTION
**Pull request by elizeugba**
I'll only be able to test this on Linux this Monday, also I don't have a Mac, so... @iagolaguna I'll leave the testing on Mac part with you (if you decide to accept this PR). 😄

### CODE REVIEW

File: src/commands/commands.js [hunk 1]
```diff
@@ -131,6 +131,60 @@ async function handler (error, callback = () => Promise.resolve()) {
...
+  if (!currentPlayback) return;
+  const increasedVolume = currentPlayback.device.volume_percent > 90
+    ? 100 : currentPlayback.device.volume_percent + 10
+  putVolume(increasedVolume)
+}
+
+async function getUserPlaylists () {
+  const { data: userPlaylistsData } = await axios.get(`${USER_PLAYLISTS_URL}`, {})
+  const basicPlaylistsData = userPlaylistsData.items.map(playlist =>
+    Object.assign({
```
Comment (iagolaguna) [inline src/commands/commands.js]: why are you  using `Object.assign`? Do you know that can you do just this ? 
```
const basicPlaylistsData = userPlaylistsData.items.map(playlist => {description: `Owner: ${playlist.owner.display_name}`, details: playlist.uri,label: playlist.name});
```

**Review by iagolaguna — COMMENTED** [review]
Hello @elizeugba , i'm very glad that you sent a PR to this project,
i made a little comment about one part of your implementation, considere review it for me approve this PR.

File: src/commands/commands.js (hunk 1 above)
Comment (elizeugba) [reply src/commands/commands.js]: I remember now that I actually tried to return using just `=> { ... }`, but I was getting an error. What I should've done was returning like this: `=> ({ ... })`. hehehe

**Comment by iagolaguna** [comment]
I see that you changed what i asked, but i let pass something very important, you forget to put the new commands in package.json, this is very important because it's a contract if the command does not exist there probably will don't work.

i think that is the only thing missing :)

**Comment by elizeugba** [comment]
I wasn't aware of that at all, I hope I covered it all with my last commit. :)

**Comment by iagolaguna** [comment]
Hello again, i test a scenario using vscodefy with a smartphone and i get an error, `{
  "error": {
    "status": 403,
    "message": "Player command failed: Cannot control device volume",
    "reason": "VOLUME_CONTROL_DISALLOW"
  }
}`
try control the volume when you are using an mobile device.
you will get the same error, but when i change to reproduce the spotify on my notebook and try change the volume i don't get the error.
Will need to do some switch when the user are using a mobile device and when the user are using a desktop device, because this feature will only work when the user are using desktop devices.


You can simulate the error here : https://developer.spotify.com/console/put-volume/
just put a sound in your phone and try change the volume through the API

Spotify has this repo for reports, bugs, quetions and suggestions, maybe you can ask there about, this
is the repo : https://github.com/spotify/web-api

Inside vscodefy constantly i ask to the api about the state of the track, because the user can modify the state through the spotify app, or another applications.
Maybe you can check if the current device is a mobile or desktop device and if is a desktop device enable the feature to up/down volume, are you think about that ?

**Comment by elizeugba** [comment]
I'll see if the "change playlist" feature has the same problem, if it doesn't, maybe I'll split them into two different PR, so I can work on this problem without holding back on the other feature.

**Pull request closed without merging**